resources/definitions/vertex_nano_k8600.def.json

```diff
@@ -0,0 +1,83 @@
+{
+    "version": 1,
```

Comment:
To be compatible with Cura 2.3 and higher, this version number needs to be `2`. Did you test this in Cura 4.1? The entire printer doesn't get added at all.

Reply:
Thanks for the comments,
Is there a reference manual to be found so I won't make the same mistake twice?